python-backend/app/api/variances/resources/variance_resource.py

```diff
@@ -0,0 +1,39 @@
+from flask_restplus import Resource
+from flask import request
+from sqlalchemy_filters import apply_pagination
+
+from app.extensions import api
+
+from ..models.variance import Variance
+from ..response_models import PAGINATED_VARIANCE_LIST
+from ...utils.access_decorators import requires_any_of, MINE_VIEW
+from ...utils.resources_mixins import UserMixin, ErrorMixin
+
+PAGE_DEFAULT = 1
+PER_PAGE_DEFAULT = 25
```

Comment:
Not suggesting you make this change, but these should probably be a common and shared constant as other endpoints hard-code 25. I'll keep in mind to do so if I'm working on something where it makes sense to do so. We have some continued back-end refactoring in our tech backlog.

Reply:
Definitely agree with that. Thanks for raising it 👍